scripts/check_network_config.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+NETWORK_CONFIG_FILE=$1
+PATH_LENGHT=$[${#NETWORK_CONFIG_FILE}-15]
+PATH_TO_CONFIG=${NETWORK_CONFIG_FILE:0:PATH_LENGHT}/.vagrant/machines
+NETWORK_CONFIG_TIME=$(stat --format=%Y $NETWORK_CONFIG_FILE)
+IS_RELEVANCE=1
+
+if [ -d $PATH_TO_CONFIG/ ]; then
```

Comment:
Please refactor if blocks above and below to change condition for opposite and handle error.
Use following style for error handling 
<pre>if [ error exists]; then
  echo "Error"
  exit 1
fi

code</pre>

but not this
<pre>if [ error does not exists] ; then
  code
else
  echo "Error"
  exit 1
fi</pre>